src/kixi/hecuba/weather/metofficeapi.clj

```diff
@@ -0,0 +1,33 @@
+(ns kixi.hecuba.weather.metofficeapi 
+  (:require [clojure.string :as str]
+            [clj-time.core :as t]
+            [clj-time.format :as f]
+            [clj-time.periodic :as tp]
+            [clojure.java.io :as io]
+            [clj-http.client :as client]))
+
+(defn pull-data [querydate querytime]
+  (->> (client/post "http://datagovuk.cloudapp.net/query"
+                    {:form-params {
+                                   :Type "Observation"
+                                   :PredictionSiteID "ALL"
+                                   :ObservationSiteID "ALL"
+                                   :Date querydate ;; dd/mm/yyyy
+                                   :PredictionTime querytime ;; 0000
+                                   }
+                     :follow-redirects true})
+       (:body)
+       (re-find #"https://datagovuk.blob.core.windows.net/csv/[a-z0-9]+.csv")
+       (slurp)))
+
+(defn save-file [file-name file-content] 
+  (spit file-name file-content))
+
+(defn run-data-pull [startdate] 
```

Comment:
`(defn pull-weather-for-dates [start end]...` ? And then change the t/now below to be end